cmd/fyne/internal/commands/package-wasm.go

```diff
@@ -12,8 +10,11 @@ import (
 )
 
 type indexData struct {
-	Name       string
-	IsReleased bool
+	WasmFile     string
```

Comment:
This file name is "package-wasm.go" should it be part of the new "package-web.go" instead?

Reply:
I kind of kept them separated as I was hoping that the maintenance of all this path would be temporary and we would get to wasm only shortly. This seems very unlikely now and I can definitively merge them into one file.